GNUmakefile

```diff
@@ -13,7 +13,7 @@ GO_TEST_CMD = $(if $(shell which gotestsum),gotestsum --,go test)
 ifeq ($(origin GOTEST_PKGS_EXCLUDE), undefined)
 GOTEST_PKGS ?= "./..."
 else
-GOTEST_PKGS=$(shell go list ./... | sed 's/github.com\/hashicorp\/nomad/./' | egrep -v "^($(GOTEST_PKGS_EXCLUDE))$$")
+GOTEST_PKGS=$(shell go list ./... | sed 's/github.com\/hashicorp\/nomad/./' | egrep -v "^($(GOTEST_PKGS_EXCLUDE))(/.*)?$$")
```

Comment:
we might also need to double check that if we set `GOTEST_PKGS`, that subpackages will be included.